### PR TITLE
docs: fixes and rewording of architecture

### PR DESCRIPTION
**Pull request by thomasten**
A few fixes and some wording simplifications of the architecture pages.

### CODE REVIEW

File: docs/docs/architecture/orchestration.md [hunk 1]
```diff
@@ -73,14 +70,14 @@ The *clusterID* uniquely identifies a cluster and can be used to [verify your cl
...
-For step-by-step instructions on how to do this, refer to [Upgrade Constellation](../workflows/upgrade.md).
+For step-by-step instructions on how to do this, refer to [Upgrade your cluster](../workflows/upgrade.md).
 
 ### Attestation of upgrades
 
-The new verification hashes (measurements) are released together with every image.
+With every new image, corresponding measurements are released.
 During an update procedure, the CLI provides the new measurements to the [JoinService](components.md#joinservice) securely.
 New measurements for an updated image are automatically pulled and verified by the CLI following the [supply chain security concept](attestation.md#chain-of-trust) of Constellation.
 The [attestation section](attestation.md#cluster-facing-attestation) describes in detail how these measurements are then used by the JoinService for the attestation of nodes.
```
Comment (thomasten) [inline docs/docs/architecture/orchestration.md:83]: Can you explain what the last sentence on this page means? (can't comment on that line)

File: docs/docs/architecture/keys.md [hunk 2]
```diff
@@ -33,9 +33,9 @@ The remote attestation statement of a Constellation cluster combines *baseID* an
 Constellation encrypts all cluster network communication using the [container network interface (CNI)](https://github.com/containernetworking/cni).
 See [network encryption](networking.md) for more details.
 
-The Cilium agent running on each cluster node will establish a secure WireGuard tunnel between it and all other known nodes in the cluster.
-Each node automatically creates its own [Curve25519](http://cr.yp.to/ecdh.html) encryption key-pair and distributes its public key via Kubernetes.
-Each node’s public key is then used by other nodes to decrypt and encrypt traffic from and to Cilium-managed endpoints running on that node.
+The Cilium agent running on each node establishes a secure [WireGuard](https://www.wireguard.com/) tunnel between it and all other known nodes in the cluster.
+Each node creates its own [Curve25519](http://cr.yp.to/ecdh.html) encryption key pair and distributes its public key via Kubernetes.
+A Node uses another node's public key to decrypt and encrypt traffic from and to Cilium-managed endpoints running on that node.
```
Comment (m1ghtym0) [inline docs/docs/architecture/keys.md:38]:
```suggestion
A node uses another node's public key to decrypt and encrypt traffic from and to Cilium-managed endpoints running on that node.
```
We have been using lowercase "node" here. Not sure if there are other parts of the documentation where we used uppercase "Node". Might make sense when we use the "Kubernetes Node" term similar to "Pod". It's a wild mix in the k8s docs as well...

**Review by m1ghtym0 — APPROVED** [review]
Great work, another quality improvement for our docs:-)

File: docs/docs/architecture/orchestration.md (hunk 1 above)
Comment (thomasten) [reply docs/docs/architecture/orchestration.md:83]:
```markdown
### Attestation of upgrades

With every new image, corresponding measurements are released.
During an update procedure, the CLI provides the new measurements to the [JoinService](components.md#joinservice) securely.
New measurements for an updated image are automatically pulled and verified by the CLI following the [supply chain security concept](attestation.md#chain-of-trust) of Constellation.
The [attestation section](attestation.md#cluster-facing-attestation) describes in detail how these measurements are then used by the JoinService for the attestation of nodes.

The updated measurements are reproducible based on the updated node images, hence,  auditable by the customer.
```
@m1ghtym0 I don't get the last sentence. Can you explain?

Comment (m1ghtym0) [reply docs/docs/architecture/orchestration.md:83]: Should say: A user can verify the new image's measurements by reproducing them locally using our reproducible image builds

Comment (thomasten) [reply docs/docs/architecture/orchestration.md:83]: Is my rephrasing ok?

File: docs/docs/architecture/keys.md (hunk 2 above)
Comment (thomasten) [reply docs/docs/architecture/keys.md]: Changed to lowercase. Uppercase was actually not on purpose.

**Review by m1ghtym0 — APPROVED** [review]

**Pull request merged**